pkg/ebpf/tracer.go

```diff
@@ -45,7 +45,8 @@ type Tracer struct {
 
 	reverseDNS network.ReverseDNS
 
-	perfMap *bpflib.PerfMap
+	perfMap      *bpflib.PerfMap
```

Comment:
Do we still need the perf.Map ? It should be all handled by PerfBatchManager right ?

Reply:
at least in the current state of things we do. the polling is still done outside the `PerfBatchManager`. I didn't want to change much the event loop around the perf polling code, but it could make sense to consolidate everything in one place

Reply:
Sorry, I left this comment and then continued reviewing and forgot to remove this comment. Leaving the perf map here makes sense. 

Having everything be in the manager would require coordinating with a second goroutine which seems like trouble.